app/src/main/res/values/button_themes.xml

```diff
@@ -9,9 +9,15 @@
     <style name="Hedvig.Button.Large">
         <item name="android:padding">@dimen/base_margin_double</item>
         <item name="shapeAppearance">?shapeAppearanceLargeComponent</item>
+        <item name="colorButton">@color/colorLargeButton</item>
+        <item name="colorPrimary">@color/colorOnPrimary</item>
+        <item name="textColorButton">@color/colorOnPrimary</item>
     </style>
 
     <style name="Hedvig.Button.Text" parent="Widget.MaterialComponents.Button.TextButton" />
 
-    <style name="Hedvig.Button.Outlined" parent="Widget.MaterialComponents.Button.OutlinedButton" />
+    <style name="Hedvig.Button.Outlined" parent="Widget.MaterialComponents.Button.OutlinedButton">
+        <item name="strokeColor">@color/colorPrimary</item>
```

Comment:
I think this is the default for outlined buttons

Reply:
True, i dont know why i added this then 🤔 ill remove it